CesiumUtility/include/CesiumUtility/CreditReferencer.h

```diff
@@ -0,0 +1,110 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <vector>
+
+namespace CesiumUtility {
+
+class CreditSystem;
+struct Credit;
+
+/**
+ * @brief Provides a way to reference a set of credits in a
+ * {@link CreditSystem} so that the references can easily be released later.
+ */
```

Comment:
Maybe add a clause like "Multiple CreditReferencers may be used on the same credit system to track separate sets of references -- e.g., two sets of credits from different frames."

That could be deduced from the way `CreditReferencer` is used, but it still helps to state the rationale in the class's own documentation.